### DISCUSSION TITLE
Mongo UUID column displayed as binary

### DISCUSSION THREAD

**Discussion by cobap** [dbeaver/dbeaver#20154] [Q&A]
Hello everyone! I'm trying to consume my mongo database through DBeaver but seems like my columns which have a uuid type (mongo properties in this case) are all seen as binary (printscreen below)

![image](https://github.com/dbeaver/dbeaver/assets/6344807/e78efdfc-51ba-4178-983f-5a30601b4eec)

My datasource consists as the image below. The properties are simple UUID objects
![image](https://github.com/dbeaver/dbeaver/assets/6344807/9bd5117e-c73a-447b-9150-1843b796a75e)

Any idea I can overcome this?

**Comment by E1izabeth** [comment]
Could you provide some minimal example to reproduce your issue?

**Reply by cobap** [reply]
Hello @E1izabeth, please take a look on a screenshot from my Mongo database. The fields are simple UUID

![image](https://github.com/dbeaver/dbeaver/assets/6344807/bd08a757-cb64-4780-861a-83b071af26b2)

**Comment by mkgrgis** [comment]
This problem is also applicable to SQLite `blob` with length = 16. And Oracle `RAW(16)`. It's also forms of UUID.

**Comment by E1izabeth** [comment]
Could you create an issue? I suppose it's a bug.

**Comment by cobap** [comment]
Bug created @E1izabeth, appreciate the support

https://github.com/dbeaver/dbeaver/issues/20513
